### ISSUE TITLE
DITAVAL with conditions for group default values

### ISSUE THREAD

**Issue by TroacaDenis** [dita-ot/dita-ot#4256]
## Expected Behavior
If we have a conditional attribute group such as `groupA(a b)` and we set the group to **exclude** in a ditaval file using `<prop action="exclude" att="audience" val="groupA"/>` it should set the default values for `a` and `b` to **exclude** and the entire group should evaluate to **exclude**, according to [DITA Specification](https://docs.oasis-open.org/dita/dita/v1.3/os/part1-base/langRef/ditaval/ditaval-val.html).

## Actual Behavior
The default values of `a` and `b` are not changed and the group evaluates to **include**.

## Steps to Reproduce

1. Download the following testcase: [test-condition.zip](https://github.com/dita-ot/dita-ot/files/12248441/test-condition.zip)
2. Publish main_map.ditamap to PDF

## Environment

* DITA-OT version: DITA-OT version:4.1
* Operating system and version:
  _(macOS Big Sur Version 11.6)_
* How did you run DITA-OT?
  _(oXygen)_
* Transformation type:
  _(PDF)_

**Comment by chrispy-snps** [comment]
@TroacaDenis - when you use a group, treat the group name as the `att` name:

```
<val>
    <prop action="exclude" att="groupA"/>
</val>
```

and I think this will give you the behavior that you want.

**Comment by raducoravu** [comment]
@chrispy-snps Denis is one of our interns, he was just testing to see if what the specification states works in the published output and according to the specs this should work as well:

      <prop action="exclude" att="audience" val="groupA"/>

**Comment by jelovirt** [comment]
Example in the linked spec page_

> `<p product="appserver(A B)">` is filtered out, because there is no explicit rule for A or B, and values in the "appserver" group inside of `@product` default to exclude.

**Comment by raducoravu** [comment]
Yes, I think @TroacaDenis 's interpretation comes from this example in the specification:

     <prop action="exclude" att="product" val="appserver"/>

     The first <prop> element excludes the value "appserver" when used within the @product attribute. It also sets a default of "exclude" for values within any appserver() group inside of the @product attribute.
